builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -66,17 +66,20 @@ node  {
 
 		events.sendStartedEvent("VALIDATE_INPUT", 'input validation starts', context_map)
 		if (service_config['type'] == "api" || service_config['type'] == "lambda" || service_config['type'] == "function" || service_config['type'] == "website") {
-			if (service_config['runtime'] == "nodejs" || service_config['runtime'] == "python" || service_config['runtime'] == "java" || service_config['runtime'] == "n/a") {
+			if (service_config['runtime'] == "nodejs" || service_config['runtime'] == "python2.7" || service_config['runtime'] == "python3.6" || service_config['runtime'] == "java" ||  service_config['runtime'] == "n/a") {
```

Comment:
This will require two things -  
1.  UI's create service panel should send this new runtime (python2.7  or python3.6) when a service is created. See https://github.com/tmobile/jazz/blob/python3/core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html#L89 
2. /create-service API should accept these two new values instead of old value (python). See https://github.com/tmobile/jazz/blob/python3/core/jazz_create-serverless-service/swagger/swagger.json#L280
3.  /services API should accept the new runtime values.  See https://github.com/tmobile/jazz/blob/python3/core/jazz_services/config/global-config.json#L112  

These changes are missing in the PR.